__tests__/index.test.js

```diff
@@ -80,3 +80,42 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("/api/articles", () => {
+  const getRequest = request(app).get("/api/articles");
+
+  test("GET:200 responds with an array", () => {
+    return getRequest.expect(200).then((response) => {
+      const { articles } = response.body;
+      expect(Array.isArray(articles)).toBe(true);
+    });
+  });
+
+  test("GET:200 responds with an array of objects with comment_count included", () => {
+    return getRequest.then((response) => {
+      const { articles } = response.body;
+      articles.forEach((article) => {
+        expect(article).toHaveProperty("comment_count");
+      });
```

Comment:
This test is great, but it would be better if we also test the other properties in the object and that the article body is not included. Have a look at `toMatchObject` which will help with this

Reply:
resolved